### PR TITLE
Changes to DSV output modules to use shared base class

### PR DESCRIPTION
**Pull request by joachimmetz**
Changes to DSV output modules to use shared base class

### CODE REVIEW

File: plaso/output/dynamic.py [hunk 1]
```diff
@@ -1,17 +1,18 @@
 # -*- coding: utf-8 -*-
-"""Contains a formatter for a dynamic output module for plaso."""
+"""Dynamic selected delimeter separated values output module."""
```
Comment (Onager) [inline plaso/output/dynamic.py:2]: Delimiter, not delimeter

File: plaso/output/formatting_helper.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+from __future__ import unicode_literals
+
+from plaso.lib import errors
+
+
+class FieldFormattingHelper(object):
+  """Output module field formatting helper."""
+
+  # Maps the name of a fields to a a callback function that formats
```
Comment (Onager) [inline plaso/output/formatting_helper.py:12]: Please rewrite this comment to something like: "Maps the name of a field to callback function that formats the field value"

**Comment by joachimmetz** [comment]
Rebased and addressed merge conflicts.

File: plaso/output/dynamic.py (hunk 1 above)
Comment (joachimmetz) [reply plaso/output/dynamic.py:2]: done

File: plaso/output/formatting_helper.py (hunk 2 above)
Comment (joachimmetz) [reply plaso/output/formatting_helper.py:12]: done

**Comment by joachimmetz** [comment]
Addressed review comments.

**Pull request merged**